src/main/scala/com/advancedtelematic/director/http/AdminResource.scala

```diff
@@ -257,6 +267,9 @@ class AdminResource(extractNamespace: Directive1[Namespace], val keyserverClient
 
   def multiTargetUpdatesRoute(ns: Namespace): Route =
     pathPrefix("multi_target_updates" / UpdateId.Path) { updateId =>
+      (pathEnd & get) {
```

Comment:
You can factor out `pathEnd`.

Reply:
I did consider it, but in this case it seems more readable like this...